### PR TITLE
DNEXT-14684 add .MD files for SECKFMGetUserInfo & SECKFMGetIDFlags.

### PR DESCRIPTION
**Pull request by shaiknayabrasool-s**

### CODE REVIEW

File: docs/reference/Func/SECKFMGetIDFlags.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+processes.\n");
+}
+else
+{
+    printf("SECKFMGetIDFlags returned ID as shareable with other processes.\n");
+    wFlags &= ~fIDFH_PWShareable;
+}
+```
+
+**See Also :**
```
Comment (raghu-madat) [inline docs/reference/Func/SECKFMGetIDFlags.md:65]: For See also Add  SECKFMGetUserInfo and other SEC APIs.

File: docs/reference/Func/SECKFMGetUserInfo.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return sStatus;
+}
+printf("SECKFMGetUserInfo returned username %s.\n", szUserName);
+SECKFMClose(&hKFC,
+            0,
+            0,
+            NULL);
+```
+
+**See Also :**
```
Comment (raghu-madat) [inline docs/reference/Func/SECKFMGetUserInfo.md:60]: For See also Add  SECKFMGetIDFlags and other SEC APIs.

**Review by raghu-madat — CHANGES_REQUESTED** [review]

File: docs/reference/Func/SECKFMGetIDFlags.md (hunk 1 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/SECKFMGetIDFlags.md:65]: Added.

File: docs/reference/Func/SECKFMGetUserInfo.md (hunk 2 above)
Comment (shaiknayabrasool-s) [reply docs/reference/Func/SECKFMGetUserInfo.md:60]: Added.

**Review by raghu-madat — APPROVED** [review]

**Review by vickum — APPROVED** [review]
looks fine.

**Review by saravananbalakris — APPROVED** [review]
Approved

**Pull request merged**